### ISSUE TITLE
arm and i586 not buildable because of Socket Timeout

### ISSUE THREAD

**Issue by skriesch** [qorelanguage/module-json#84]
I wanted to update module-json for the latest qore version.
arm and i586 are failing because of the following error message:
```

[   46s] Ran 2 test cases, 2 succeeded (6 assertions)
[   46s] + qore -l ./json-api-1.3.qmod test/JsonRpcHandler.qtest -v
[  946s] QUnit Test "JsonRpcHandlerTest" v1.0
[  946s] ERROR: Test JSON-RPC 2.0: 0 assertions
[  946s] Unexpected exception: SOCKET-TIMEOUT at test/JsonRpcHandler.qtest:148 (Qore)
[  946s] -----
[  946s] 	>> System exception: test/JsonRpcHandler.qtest:148 (Qore): SOCKET-TIMEOUT: timed out after 300000 milliseconds in Socket::recv() call
[  946s] 	>>   JsonRpcClient::call() called at test/JsonRpcHandler.qtest:148 (Qore builtin code)
[  946s] 	>>   JsonRpcHandlerTest::testJsonRpc2_0() called at <builtin>:-1 (Qore user code)
[  946s] 	>>   call_function_args() called at /usr/share/qore-modules/1.12.4/QUnit.qm:507 (Qore builtin code)
[  946s] 	>>   TestCase::run() called at /usr/share/qore-modules/1.12.4/QUnit.qm:2120 (Qore user code)
[  946s] 	>>   Test::main() called at test/JsonRpcHandler.qtest:115 (Qore user code)
[  946s] 	>>   JsonRpcHandlerTest::constructor() called at <builtin>:-1 (Qore user code)
[  946s] -----
[  946s] ERROR: Test JSON-RPC 1.0: 0 assertions
[  946s] Unexpected exception: SOCKET-TIMEOUT at test/JsonRpcHandler.qtest:172 (Qore)
[  946s] -----
[  946s] 	>> System exception: test/JsonRpcHandler.qtest:172 (Qore): SOCKET-TIMEOUT: timed out after 300000 milliseconds in Socket::recv() call
[  946s] 	>>   JsonRpcClient::call() called at test/JsonRpcHandler.qtest:172 (Qore builtin code)
[  946s] 	>>   JsonRpcHandlerTest::testJsonRpc1_0() called at <builtin>:-1 (Qore user code)
[  946s] 	>>   call_function_args() called at /usr/share/qore-modules/1.12.4/QUnit.qm:507 (Qore builtin code)
[  946s] 	>>   TestCase::run() called at /usr/share/qore-modules/1.12.4/QUnit.qm:2120 (Qore user code)
[  946s] 	>>   Test::main() called at test/JsonRpcHandler.qtest:115 (Qore user code)
[  946s] 	>>   JsonRpcHandlerTest::constructor() called at <builtin>:-1 (Qore user code)
[  946s] -----
[  946s] ERROR: Test JSON-RPC 1.1: 0 assertions
[  946s] Unexpected exception: SOCKET-TIMEOUT at test/JsonRpcHandler.qtest:196 (Qore)
[  946s] -----
[  946s] 	>> System exception: test/JsonRpcHandler.qtest:196 (Qore): SOCKET-TIMEOUT: timed out after 300000 milliseconds in Socket::recv() call
[  946s] 	>>   JsonRpcClient::call() called at test/JsonRpcHandler.qtest:196 (Qore builtin code)
[  946s] 	>>   JsonRpcHandlerTest::testJsonRpc1_1() called at <builtin>:-1 (Qore user code)
[  946s] 	>>   call_function_args() called at /usr/share/qore-modules/1.12.4/QUnit.qm:507 (Qore builtin code)
[  946s] 	>>   TestCase::run() called at /usr/share/qore-modules/1.12.4/QUnit.qm:2120 (Qore user code)
[  946s] 	>>   Test::main() called at test/JsonRpcHandler.qtest:115 (Qore user code)
[  946s] 	>>   JsonRpcHandlerTest::constructor() called at <builtin>:-1 (Qore user code)
```

Reproducible[ link](https://build.opensuse.org/package/live_build_log/home:AdaLovelace:branches:devel:languages:misc/qore-json-module/openSUSE_Factory/i586)

**Comment by davidnich** [comment]
@skriesch this may not be properly documented anywhere, but we open all issues with Qore modules in the qore issue tracker - this issue has been fixed already for Qore 1.13: https://github.com/qorelanguage/qore/issues/4660 - will be released this week...

**Comment by skriesch** [comment]
Thank you for this hint. I have chosen this project, because this exception has happened in the module-json.
